README.rst

```diff
@@ -15,26 +15,27 @@ django-freeradius
    :target: http://badge.fury.io/py/django-freeradius
 
 ------------
+Django-freeradius is part of the `OpenWISP project <http://openwrt.org>`_.
 
-TODO:
+.. image:: http://netjsonconfig.openwisp.org/en/latest/_images/openwisp.org.svg
+  :target: http://openwisp.org
+
+**django-freeradius** is `Django <https://www.djangoproject.com/>`_ reusable app that provides an admin interface to a `freeradius <http://freeradius.org/>`_ database.
 
-------------
 
 .. contents:: **Table of Contents**:
    :backlinks: none
    :depth: 3
 
 ------------
 
-Current features
-----------------
-
-* TODO
-
 Project goals
 -------------
 
-* TODO
+* provide a web interface to manage a freeradius database, with an additional RESTful API
```

Comment:
remove the mention of the RESTful API, we'll add that when we'll have implemented it